docs/user-guide/cli-using-understand-profiles-configs.md

```diff
@@ -32,22 +34,22 @@ All configuration files are saved in `.json` format.
 
 ## How configuration files and profiles work together
 
-There may be instances where a user has all four types of files in their system, and all four configurations are referred to simultaneously by Zowe CLI for a particular command. **[is "simultaneously" correct?]**
+There may be instances where a user has all four types of files in their system, and all four configurations are referred to simultaneously by Zowe CLI for a particular command.
 
-This can mean working with files that have contradictory configurations. One file can specify that a certain profile property has a value of `ABC`, while another file uses `XYZ` as a value.
+This can mean working with files that have conflicting configurations. One file can specify that a certain profile property has a value of `ABC`, while another file uses `XYZ` as a value.
 
 When the same properties have different values across multiple configuration files, Zowe CLI follows a two-step check to determine which configurations apply:
 
-1. Does the configuration file have a more *specialized* or more *generalized* use?
-2. Is the configuration file *more specific* or *less specific*.
+1. Does the configuration file have a *more narrow* or a *more broad* scope?
+2. Is the configuration file *more specific* or *less specific*?
 
-Zowe CLI considers a user configuration file to have a more specific use than a team configuration file, and a project configuration file to be more specialized than a global configuration file, which is more generalized.
+Zowe CLI considers a user configuration file to have a more specific use than a team configuration file, and a project configuration file have a narrower scope than a global configuration file, which has a broader scope.
 
 When checking all possible configuration file types, Zowe CLI categorizes files in the manner below:
 
 ![Zowe CLI Config File Table](../images/common/cli-config-file-table.jpg)
 
-This order is applied no matter the directory in which you issue a Zowe CLI command. As a user, it can be easy to trace this logic when configuration files are all either in your home directory (i.e., specialized) or your project directory (i.e., generalized).
+This order is applied no matter the directory in which you issue a Zowe CLI command. As a user, it can be easy to trace this logic when configuration files are all either in your ZOWE_CLI_HOME directory (i.e., narrow scope) or your project directory (i.e., broad scope).
```

Comment:
I think the config file in ZOWE_CLI_HOME would be broad scope and the config file in the project directory would be narrow scope.

Reply:
Fixed - thank you!